okapi-core/src/main/java/org/folio/okapi/managers/ModuleManager.java

```diff
@@ -138,6 +138,11 @@ void enableAndDisableCheck(Tenant tenant,
       for (ModuleDescriptor md : modlist) {
         mods.put(md.getId(), md);
       }
+      String deps = DepResolution.checkAllDependencies(mods);
+      if (!deps.isEmpty()) {
+        fut.handle(new Success<>()); // failures even before we change enabled modules
```

Comment:
Ignoring pre-existing dependency issue and returning OK are reasonable when deleting modules? Would it be an issue when enabling modules?

Reply:
Yes.. How about we just only allow this when removing a module?